tests/src/test/java/alluxio/shell/command/CopyFromLocalCommandTest.java

```diff
@@ -406,4 +406,14 @@ public void copyFromLocalWildcard() throws IOException {
     Assert.assertTrue(fileExists(new AlluxioURI("/testDir/foobar3")));
     Assert.assertFalse(fileExists(new AlluxioURI("/testDir/foobar4")));
   }
+
+  @Test
+  public void copyFromLocalRelativePath() throws IOException {
+    File localDir = new File("localDir");
```

Comment:
I'm worried about writing to a local directory and polluting the tester's filesystem. To avoid this, you can use 
```java
  @Rule
  public TemporaryFolder mTestFolder = new TemporaryFolder();
```
to create a temporary folder that will be automatically cleaned up at the end of the test, then wrap the test in
```java
  try (SetAndRestoreSystemProperty s = new SetAndRestoreSystemProperty("user.dir", mTestFolder.getRoot()) {
    // test code
  }
```
so that you can test relative paths inside the test folder.

You can find examples of using `TemporaryFolder` in `AllocatorFactoryTest`, and examples of using `SetAndRestoreSystemProperty` in `ConfigurationTest`.